Woof/Woof/Model/Rating/Rating.swift

```diff
@@ -0,0 +1,8 @@
+/// Represents a rating that is used to rate something or someone.
+enum Rating: Equatable {
+    /// The rating is unavailable.
+    case unavailable
+
+    /// The rating has a specific value.
+    case rated(Double)
```

Comment:
Do we use `.rated` case in the `FiveStarRatingView`?
If so, why we have the `Double` associated value here?

Shouldn't we create a `FiveStarRating` enum that contains only cases we can have in the view?

Reply:
Yes, you're right, thanks!
Fixed.